### DISCUSSION TITLE
CustomStoragePath funktioniert nicht (für Diagramme) nach Update auf raspmatic_rpi3 (3.69.6.20230407) mit eingestecktem USB-Stick

### DISCUSSION THREAD

**Discussion by rcccbs** [OpenCCU/OpenCCU#2296] [🇩🇪 Fragen&Antworten]
Ich nutze den CustomStoragePath in Verbindung mit der rc.prelocal. Historisch bedingt (früher für ccu-historian und Diagramme) habe ich noch einen USB-Stick  in meinem Raspberry Pi 3 Model B Rev 1.2 (rpi3) eingesteckt. Seit dem Update auf die o.g.  Version erhielt ich  beim Aufruf der Diagramme die Meldung, dass kein USB Stick vorhanden sei. NAS war gemountet und ccu-historian hat auch dort hingeschrieben. Die Diagramme funktionierten nicht.  Bei meiner Suche nach einer Lösung bin ich auf Issue 1377 gestoßen, in dem auch die grundsätzliche Nutzung eines USB Sticks bei Nutzung des CustomStoragePath angesprochen wird. [https://github.com/jens-maus/RaspberryMatic/issues/1377#issuecomment-898872757](https://github.com/jens-maus/RaspberryMatic/issues/1377#issuecomment-898872757)
Nach entfernen des USB-Sticks landen die Diagrammdaten wieder auf dem NAS.
Hat sonst noch jemand dieses Verhalten beobachtet.

**Comment by Baxxy13** [comment]
Ich hatte das gestern kurz getestet und konnte dabei keine Probleme erkennen.
Du musst also ein paar Details liefern.
- Screenshot oder Text was bei CustomStoragePath eingetragen ist.
- Ausgabe (ssh Konsole) von:
`mount`
`ls -al /media`

Die Befehle jeweils im korrekten und fehlerhaften Zustand ausführen.

**Comment by rcccbs** [comment]
Habe gerade den alten usb-Stick im laufenden Betrieb eingesteckt. wurde korrekt erkannt. Hatte auch keine Auswirkungen auf  die Diagramme oder Historian. CustomStoragePath geht nach /mnt
Hier das Ergebnis vom ls -al /media 
drwxrwxrwt   11 root     root           240 Apr 18 14:52 .
drwxr-xr-x   24 root     root          4096 Apr  7 08:24 ..
drwxrwxrwx    2 root     root             0 Nov 24  2020 nas
lrwxrwxrwx    1 root     root             4 Apr 18 14:52 usb0 -> /mnt
drwxr-xr-x    5 root     root          4096 Jan  1  1970 usb1
drwxrwxr-x    2 root     root            40 Apr 18 14:51 usb2
drwxrwxr-x    2 root     root            40 Apr 18 14:51 usb3
drwxrwxr-x    2 root     root            40 Apr 18 14:51 usb4
drwxrwxr-x    2 root     root            40 Apr 18 14:51 usb5
drwxrwxr-x    2 root     root            40 Apr 18 14:51 usb6
drwxrwxr-x    2 root     root            40 Apr 18 14:51 usb7
drwxrwxr-x    2 root     root            40 Apr 18 14:51 usb8

Hatte mir gedacht dass ich den Fehler nun durch einen Reboot wieder herstellen könnte. Leider nicht. Also alle Aufregung umsonst. Ich muss aber noch dazu erwähnen, dass ich bei meiner Fehlersuche nach dem Update sowohl unter /mnt als auch auf dem Stick den Inhalt des directories measurement gelöscht habe.
Ich könnte jetzt noch ein Backup einspielen um zu versuchen das nochmals nachzustellen. Aber ich denke ich verbuche das jetzt einmal unter "Einzelschicksal". Danke für eure Unterstützung
